roles/openshift_hosted_templates/files/v3.10/enterprise/registry-console.yaml

```diff
@@ -25,6 +25,7 @@ objects:
           labels:
             name: "registry-console"
         spec:
+          nodeSelector: ${{NODE_SELECTOR}}
```

Comment:
Should be single curl braces.

Reply:
This is important, it means it a json/yaml, not string - see https://docs.openshift.org/latest/dev_guide/templates.html#writing-parameters